platoon_strategic_IHP/include/platoon_config_ihp.h

```diff
@@ -0,0 +1,89 @@
+#pragma once
+
+/*
+ * Copyright (C) 2021 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/*
+ * Developed by the UCLA Mobility Lab, 10/20/2021. 
+ *
+ * Creator: Xu Han
+ * Author: Xu Han, Xin Xia, Jiaqi Ma
+ */
+
+#include <iostream>
+
+/**
+ * \brief Stuct containing the algorithm configuration values for the YieldPluginConfig
+ */
+struct PlatoonPluginConfig
+{
+  // following parameters are for general platooning plugin
+  double vehicleLength         = 5.0;  // m
+  int    maxPlatoonSize        = 10;   // 1
+  int    algorithmType         = 0;    // N/A
+  int    statusMessageInterval = 100;  // ms
+  int    infoMessageInterval   = 200;  // ms
+  double mvr_duration          = 15;   // s
+  double epislon              = 0.001; // m/s
+  
+  // following parameters are for platoon forming and operation
+  double timeHeadway           = 2.0;  // s
```

Comment:
You might consider renaming either timeHeadway or standStillHeadway, as they have different units of measure.  The term "headway" should be consistently used as either a time or a distance.  Using it as both is a recipe for a unit conversion error somewhere.